pages/04.applications/99.wishlist/apps_wishlist.md

```diff
@@ -196,6 +196,8 @@ You can [contribute to this list by adding something you'd like to be packaged](
 | [Nautilus](https://myactivity.stream/) | Turn your website into an ActivityPub profile  | [Upstream](https://github.com/aaronpk/Nautilus) |  |
 | Netrunner |  | [Upstream](https://github.com/mtgred/netrunner) |  |
 | [Netbird](https://netbird.io/) | Create an overlay peer-to-peer network connecting machines regardless of their location | [Upstream](https://github.com/netbirdio/netbird) | |
+| [nostr] |  |  (https://nostr.com/) | The simplest open protocol that is able to create a censorship-resistant global "social" network once and for all.
+| [Upstream] (https://github.com/nostr-protocol/nostr) ||
```

Comment:
```suggestion
| [nostr] |  |  (https://nostr.com/) | The simplest open protocol that is able to create a censorship-resistant global "social" network once and for all. | [Upstream] (https://github.com/nostr-protocol/nostr) | |
```

Reply:
Yes, you're right, it doesn't sound really good, i copied it from nostr repo.

Reply:
Fixed

Reply:
This is a temporary account, i log out, bye.

Reply:
I was merely correcting the code (one-liner, and space for the last empty column). :)